README.md

```diff
@@ -1462,6 +1462,52 @@ jobs:
           publish-summary: false
 ```
 
+### Automatic PR number & URL detection
+
+When recording runs to Cypress Cloud, the PR number and URL can be automatically detected if you pass `GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}` 
+via the workflow `env`. When set, this value enables the Action to perform additional logic that grabs the related PR number and URL (if they
+exist) and sets them in the environment variables `CYPRESS_PULL_REQUEST_ID` and `CYPRESS_PULL_REQUEST_URL`, respectively.
+* See Cypress' documentation on [CI Build Information](https://docs.cypress.io/guides/continuous-integration/introduction#CI-Build-Information)
+
+Example workflow using the variables:
+```yml
+name: Example echo PR number and URL
+on: push
+jobs:
+  cypress-run:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Cypress run
+        uses: cypress-io/github-action@v5
+        with:
+          record: true
+      - run: echo "PR number is $CYPRESS_PULL_REQUEST_ID"
+      - run: echo "PR URL is $CYPRESS_PULL_REQUEST_URL"
+    env:
+      CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+```
+
+#### Branch with PR
```

Comment:
Rather than focusing these sections around branches/PRs, it might make more sense to specify the behavior based on the various workflow event types.

For example, if the action executes on a `push` event, we'll still need to make the request, even if there is a PR open, as the GITHUB_HEAD_REF env will not be set for that event.

We could do something like:

```yaml
<example yaml here>

The behavior of the action varies based on the triggering event type.

#### Triggering event: `pull_request`/`pull_request_target`

...

#### Triggering event: `push`

...
```

Reply:
Good call! I'll update this 👍 That's much clearer.